build.gradle

```diff
@@ -184,18 +188,19 @@ subprojects {
             if (file(xsltFile).exists() && file(schemaBaseDir).exists()) {
                 logger.lifecycle("Transforming Schemas")
                 ant.xslt(
-                        extension: '.xsd',
-                        basedir: schemaBaseDir,
-                        includes: "**/*.xsd",
-                        style: xsltFile,
-                        destdir: "${outputDir}"
+                    extension: '.xsd',
+                    basedir: schemaBaseDir,
+                    includes: "**/*.xsd",
+                    style: xsltFile,
+                    destdir: "${outputDir}"
                 )
             } else {
                 logger.lifecycle("NOT Transforming Schemas")
                 copy {
                     from schemaBaseDir
                     into outputDir
                     include "**/*.xsd"
+                    exclude "**/examples"
```

Comment:
were you having a problem with these?

Reply:
They were just making the build a little noisy since a few of them had an `.xsd` in them that we didn't actually want transformed.

Reply:
weird, i wonder why?

Reply:
It worked and handled it just fine, it was just complaining every time it happened. I was just trying to quite some of the noise so I could hone in on what was actually problems.